test/functional/tests/io_class/test_io_class_core_id.py

```diff
@@ -169,21 +169,21 @@ def prepare(filesystem, cores_number):
     ioclass_config.add_ioclass(
         ioclass_id=0,
         eviction_priority=22,
-        allocation=True,
+        allocation="1.00",
         rule="unclassified",
         ioclass_config_path=ioclass_config.default_config_file_path,
     )
     ioclass_config.add_ioclass(
         ioclass_id=1,
         eviction_priority=22,
-        allocation=True,
+        allocation="0.00",
```

Comment:
shouldn't it be "1.00"?

Reply:
Few lines earlier is following comment:
```
# To make test more precise all workload except of tested ioclass should be
# put in pass-through mode
```
I guess it was a bag introduced when the test was developed. I verified that is still passing with this change.